pl.art.lach.slawek.apps.v2blankbrowser.json

```
@@ -0,0 +1,28 @@
+{
+  "id": "pl.art.lach.slawek.apps.V2BlankBrowser",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "40",
+  "sdk": "org.gnome.Sdk",
+  "command": "V2BlankBrowser",
+  "clear": [ "*.o"],
+  "modules": [
+  {
+    "name": "main",
+    "buildsystem" : "cmake",
```

```suggestion
    "buildsystem" : "cmake-ninja",
```

It's faster, and prefered unless it doesn't work (which in that case is a bug)